xds/internal/xdsclient/transport/transport.go

```diff
@@ -412,7 +422,7 @@ func (t *Transport) sendExisting(stream adsStream) bool {
 
 	for url, resources := range t.resources {
 		if err := t.sendAggregatedDiscoveryServiceRequest(stream, mapToSlice(resources), url, t.versions[url], "", nil); err != nil {
-			t.logger.Warningf("ADS request failed: %v", err)
+			t.logger.Warningf("Sending ADS request for resources: %q, url: %q, version: %q, nonce: %q failed: %v", resources, url, t.versions[url], "", err)
```

Comment:
same here 
I had a comment above for the err returned from sendAggregatedDiscoveryServiceRequest. Considering that, should we add a DebugF log here which logs pretty.ToJSON(resp)?

Reply:
Replied to the previous comment. Thanks.